File: Java/Petshop.java
class Petshop {
    Petshop(){
    }
    
    private String ID_produk;
    private String nama_produk;
    private String kategori;
    private int harga;

    Petshop(String id, String nama, String Kategori, int harga){
        this.ID_produk = id;
        this.nama_produk = nama;
        this.kategori = Kategori;
        this.harga = harga;
    }

    void setId(String id){
        this.ID_produk = id;
    }

    void setNama(String nama){
        this.nama_produk = nama;
    }

    void setKategori(String kategori){
        this.kategori = kategori;
    }

    void setHarga(int harga){
        this.harga = harga;
    }

    String getId(){
        return this.ID_produk;
    }

    String getNama(){
        return this.nama_produk;
    }

    String getKategori(){
        return this.kategori;
    }

    int getHarga(){
        return this.harga;
    }
}

File: PHP/Petshop.php
<?php
class Petshop {
    private $ID_produk;
    private $nama_produk;
    private $kategori;
    private $harga;
    private $foto;

    public function __construct($ID_produk = "", $nama_produk = "", $kategori = "-", $harga = 0, $foto = "") {
        $this->ID_produk = $ID_produk;
        $this->nama_produk = $nama_produk;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    public function setId($id) {
        $this->ID_produk = $id;
    }

    public function setNama($nama) {
        $this->nama_produk = $nama;
    }

    public function setKategori($kategori) {
        $this->kategori = $kategori;
    }

    public function setHarga($harga) {
        $this->harga = $harga;
    }

    public function setFoto($foto) {
        $this->foto = $foto;
    }

    public function getId() {
        return $this->ID_produk;
    }

    public function getNama() {
        return $this->nama_produk;
    }

    public function getKategori() {
        return $this->kategori;
    }

    public function getHarga() {
        return $this->harga;
    }
    public function getFoto() {
        return $this->foto;
    }
}
?>
